### PR TITLE
Initial gradients import at color tokens

### PR DESCRIPTION
**Pull request by PetmePant**
As the title describes, this is the initial release of gradients into our color tokens.

### CODE REVIEW

**Comment by PetmePant** [comment]
@matthprost @fabienhebert check please, we need this before you work on the banner component

File: tokens.json [hunk 1]
```diff
@@ -1502,6 +1502,42 @@
...
+      "gradients": {
+        "background": {
+          "primary": {
+            "value": "$other.gradients.primary",
+            "type": "color"
+          },
+          "blue": {
+            "value": "{other.gradients.blue}",
+            "type": "color"
+          },
```
Comment (matthprost) [inline tokens.json:1515]: there is an issue here 2 different type of variable we need to keep with `$other.gradient.primary`

**Review by matthprost — CHANGES_REQUESTED** [review]

**Comment by PetmePant** [comment]
@matthprost shouldn't this be merged?

File: tokens.json [hunk 2]
```diff
@@ -1819,6 +1855,40 @@
...
+        "aqua": {
+          "value": "linear-gradient(199.45deg, #03D5E0 14.03%, #4F0599 88.52%);",
+          "type": "color"
+        },
+        "fuschia": {
+          "value": "linear-gradient(200.02deg, #FF1C6E 13.34%, #4F0599 86.64%);",
+          "type": "color"
+        },
+        "orange": {
+          "value": "linear-gradient(201.62deg, #FF5D2A 14.62%, #4F0599 85.81%);",
```
Comment (matthprost) [inline tokens.json:1889]: Possible not to have `;` at the end of them?

**Review by matthprost — CHANGES_REQUESTED** [review]

File: tokens.json (hunk 2 above)
Comment (PetmePant) [reply tokens.json]: just pushed a new request, can you check ?

**Review by matthprost — APPROVED** [review]

**Pull request merged**